axelrod/plot.py

```diff
@@ -59,3 +61,33 @@ def payoff(self):
             return figure
         else:
             return None
+
+    def stackplot(self, populations):
+        if self.matplotlib_installed:
+            figure, ax = plt.subplots()
+            turns = range(len(populations))
+            pops = [[populations[iturn][ir] for iturn in turns] for ir in self.result_set.ranking]
+            ax.stackplot(turns, *pops)
+
+            ax.yaxis.tick_right()
+            ax.yaxis.set_label_position("right")
+            plt.ylim([0.0, 1.01])
+            plt.ylabel('Relative population size')
+            plt.xlabel('Turn')
+            plt.title("Strategy population dynamics based on average payoffs")
+            trans = transforms.blended_transform_factory(ax.transAxes, ax.transData)
```

Comment:
Transforms in matplotlib allow one to do nonstandard operations on various coordinate systems and to combine them. A tutorial: http://matplotlib.org/users/transforms_tutorial.html. In this case I used it to create extra tick marks on the left side for the strategy names.